compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -74,9 +74,12 @@ const Tensor *KernelBuilder::getInputTensor(const loco::Node *node) const
   return tensor;
 }
 
-const Tensor *KernelBuilder::getOptionalInputTensor(const loco::Node *node) const
+const Tensor *KernelBuilder::getOptionalInputTensor(loco::Node *node) const
 {
-  // TODO Revise this when optional inputs are implemented in the IR.
+  if (dynamic_cast<luci::CircleOutputExclude *>(node))
```

Comment:
It is not necessary to remove `const` on the parameter if apply the suggested change:
```suggestion
  if (dynamic_cast<const luci::CircleOutputExclude *>(node))
```

Reply:
oh! Thanks I will make a change and make a official PR for it.